File: desarrollo/FrontEnd/src/app/ViewModels/HotelFilterQueryParametersViewModel.ts
import { PaginationOptions } from "../shared/PaginationOptions";

export class HotelFilterQueryParametersViewModel{
    FilterName? : string;
    FilterTop? : boolean;
    PagOptions : PaginationOptions; 
    
    constructor(PagOptions:PaginationOptions, FilterName?:string, FilterTop?:boolean ){
        this.FilterName = FilterName,
        this.FilterTop = FilterTop,
        this.PagOptions = PagOptions
    }
}

File: desarrollo/Backend/PruebaPlayaSenator/PruebaPlayaSenator.Application/ViewModel/HotelFilterQueryParametersViewModel.cs
using PruebaPlayaSenator.Application.Shared;
using System;
using System.Collections.Generic;
using System.Text;

namespace PruebaPlayaSenator.Application.ViewModel
{
    public class HotelFilterQueryParametersViewModel
    {
        public string FilterName { get; set; }
        public bool? FilterTop { get; set; }
        public PaginationOptions PagOptions { get;set;}
    }
}
